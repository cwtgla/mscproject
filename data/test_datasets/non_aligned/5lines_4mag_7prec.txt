15.127
8.1
0.009
-3.005
-15.127