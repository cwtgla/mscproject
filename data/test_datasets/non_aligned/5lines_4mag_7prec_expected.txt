127
244
100
0
153
133
255
240